package.json

```diff
@@ -98,6 +98,8 @@
     "defaults"
   ],
   "resolutions": {
+    "boxen": "^5.1.2",
+    "eslint": "^7.17.0",
```

Comment:
These were installing multiple times, and causing a set of typing issues.

Also installing less versions of eslint during development seems like a valid plan to me